src/live.html

```diff
@@ -130,28 +130,34 @@ <h1 id="title">Live</h1>
 		<nav>
 			<ul>
 				<li data-bind-view="dayof">
-					<a href="#/dayof">Home</a>
+					<a href="#/dayof">Day of</a>
+				</li>
+				<li data-bind-view="live">
+					<a href="#/live">Live</a>
 				</li>
 				<li data-bind-view="schedule">
 					<a href="#/schedule">Schedule</a>
 				</li>
-				<li data-bind-view="travel">
-					<a href="#/travel">Travel</a>
-				</li>
 				<li>
 					<a href="http://maps.hackupc.com/#/UPC" target="_blank">Maps</a>
 				</li>
+				<li>
+					<a href="https://mentors.hackupc.com/" target="_blank">Mentors</a>
+				</li>
 				<li data-bind-view="hardware">
-					<a href="#/hardware">Hardware</a>
+					<a href="#/hardware">Hardware List</a>
 				</li>
 				<li data-bind-view="challenges">
-					<a href="#/challenges">Challenges</a>
+					<a href="#/challenges">Challenges & Prizes</a>
```

Comment:
The title is too long and it takes two lines to display. The navbar looks too wide as a consequence
<img width="1440" alt="screen shot 2017-10-13 at 00 27 40" src="https://user-images.githubusercontent.com/9166418/31522291-5e61b5d0-afad-11e7-80e6-caef049c77d6.png">